### DISCUSSION TITLE
Fakebook source code

### DISCUSSION THREAD

**Discussion by rbgfloripa** [remix-run/remix#4586] [Q&A] [answered]
HI all,

Is the source code of the "Fakebook" application from the remix website available somewhere ??

Thanks

Rafael.

**Comment by machour** [comment, marked as answer]
Hi, check this version: https://github.com/kentcdodds/fakebooks-remix

**Reply by rbgfloripa** [reply]
Hi,

yes I saw this code
But it doesn't look official to me.
Sounds like someone looked at the site and tried to do something similar.

**Reply by machour** [reply]
(Kent worked at Remix and is quite good at it, so I guess the code would be approximately what's on the original app)
I'll look into having the original app added in the examples repo if that's ok with the team 👍🏼

**Reply by rbgfloripa** [reply]
Thanks a lot !
